pyo3-ffi/src/object.rs

```diff
@@ -740,9 +790,39 @@ pub unsafe fn Py_XNewRef(obj: *mut PyObject) -> *mut PyObject {
     obj
 }
 
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_NONE: c_uint = 0;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_FALSE: c_uint = 1;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_TRUE: c_uint = 2;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_ELLIPSIS: c_uint = 3;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_NOT_IMPLEMENTED: c_uint = 4;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_ZERO: c_uint = 5;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_ONE: c_uint = 6;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_EMPTY_STR: c_uint = 7;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_EMPTY_BYTES: c_uint = 8;
+#[cfg(Py_3_13)]
+pub const Py_CONSTANT_EMPTY_TUPLE: c_uint = 9;
+
+extern "C" {
+    #[cfg(Py_3_13)]
+    #[cfg_attr(PyPy, link_name = "PyPy_GetConstant")]
```

Comment:
I guess there's no harm and it will probably work this way, but it seems a little premature to add code to support PyPy 3.13, which won't be coming out for quite a while, if ever.

Reply:
Yes I have gone back and forth on these "future" bindings for PyPy, but I've also had so many cases where the `link_name` attribute has randomly missing creating compile errors for users on PyPy I just think it's slightly better this way round now.